torch/csrc/jit/codegen/cuda/scheduler/reduction_heuristic.h

```diff
@@ -111,7 +115,7 @@ class ReductionParams {
     bool attr_equal = other.fastest_dim == fastest_dim &&
         other.persistent_kernel == persistent_kernel &&
         other.project_persistent_buffers == project_persistent_buffers &&
-        other.schedule_3D == schedule_3D &&
```

Comment:
flip_grid was never used, may want to leave it in to play with later because I didn't test in the end if there are any cases that can benefit. Just please mark with a TODO.